app/models/user/User.scala

```diff
@@ -203,11 +204,25 @@ class UserDAO @Inject()(sqlClient: SqlClient)(implicit ec: ExecutionContext)
     // format: off
     UserCompactInfo(<<[ObjectId],<<[ObjectId],<<[String],<<[String],<<[String],<<[String],<<[Boolean],<<[Boolean],
       <<[Boolean],<<[Boolean],<<[String],<<[String],<<[String],<<[String], <<[String],<<[Instant],
-      <<[String],<<[String],<<[String],<<[Instant],<<?[String],<<[Boolean],<<[Boolean],<<[Boolean]
+      <<[String],<<[String],<<[String],<<[Instant],<<?[String],<<[Boolean],<<[Boolean],<<[Boolean],<<[Boolean]
     )
     // format: on
   }
 
+  private def payingOrganizationInfoSubquery =
+    q"""(
+         SELECT DISTINCT ON (_multiUser) _multiUser, _organization
+            FROM webknossos.users
+            WHERE NOT isDeactivated
+              AND _organization IN (
+                SELECT _id
+                FROM webknossos.organizations
+                WHERE pricingPlan IN ('Team', 'Power', 'Custom')
+              )
+            ORDER BY _multiUser, created ASC
+         ) AS payingOrganization ON payingOrganization._multiUser = u._multiUser
+     """
```

Comment:
This subquery can only be used in the context of a JOIN. In fact, I see only LEFT JOIN usages. Maybe pull that in here? Right now, the bare ON makes this kind of “half a unit”, but you only notice that when reading the last line (or the usages)

Alternatively, maybe remove the `AS payingOrganization ON payingOrganization._multiUser = u._multiUser` and move it to the usages. That may actually make them more readable, and this subquery could stand on its own.